collect_app/src/androidTest/java/org/odk/collect/android/feature/formmanagement/MatchExactlyTest.kt

```diff
@@ -223,4 +223,24 @@ class MatchExactlyTest {
             .clickFillBlankForm()
             .assertFormDoesNotExist("One Question Updated")
     }
+
+    @Test
+    fun whenMatchExactlyStopped_showNotification_andOpenFormListOnClick() {
```

Comment:
Nice! This is really clean. It makes me realize that we'd also want a test for the case where the job is stopped but completes successfully (we shouldn't see a notification in that scenario).